src/components/dashboard/Rewards.web.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react'
+import userStorage from '../../lib/gundb/UserStorage'
+import logger from '../../lib/logger/pino-logger'
+const log = logger.child({ from: 'RewardsTab' })
+
+const RewardsTab = props => {
+  const [loginToken, setLoginToken] = useState()
+  const getToken = async () => {
+    let token = await userStorage.getProfileFieldValue('loginToken')
+    token = 'x'
+    log.debug('got rewards login token', token)
+    setLoginToken(token)
+  }
+  useEffect(() => {
+    getToken()
+  }, [])
+
+  return loginToken ? (
+    <div
+      style={{
+        overflow: 'scroll !important',
+        WebkitOverflowScrolling: 'touch !important',
+      }}
+    >
+      <iframe
+        title="Rewards"
+        src={`https://w3.gooddollar.org?loginToken=${loginToken}`}
```

Comment:
@sirpy the correct name of query parameter for login token is just 'token' not 'loginToken'